doc/langref.html.in

```diff
@@ -9463,9 +9463,92 @@ test "assert in release fast mode" {
       </p>
       {#header_close#}
       {#header_open|Zig Build System#}
-      <p>TODO: explain purpose, it's supposed to replace make/cmake</p>
-      <p>TODO: example of building a zig executable</p>
-      <p>TODO: example of building a C library</p>
+
+      <p>Zig's build system is intended to replace existing tools
+      like Make and CMake. It introduces no dependencies, so Zig code
```

Comment:
The docs should explain the purpose without relying on external knowledge about what Make and CMake are. No mention of Make or CMake here

Reply:
I expanded this section considerably and removed mention of other tools. It feels weird not to acknowledge their existence, but I can see how this might not be the place for that.